### ISSUE TITLE
Detach camera from display dependency (v831)

### ISSUE THREAD

**Issue by ricardojlrufino** [sipeed/MaixPy3#39]
I was able to successfully run an old version of MaixPy3 on a  generic IP camera (v831).
However, the new versions of the library (camera) seem to be strongly coupled to the need to have a display.

It would be interesting to have some parameter in `cam.config(size, display=True)`, which would serve to activate or not the display.

```
from maix import camera
camera.config(size=(640, 480), display=False)
```

**Comment by junhuanchen** [comment]
Indeed, in my latest version, I thought about it and turned off the default display on the V831, which was originally designed as an extension of the camera app because of its multilayer device (VO) support.
